main.js

```diff
@@ -0,0 +1,85 @@
+// the game board to display with the 1st peg being the "start" peg
+let board = [
+  [5,4,3,2,1],
+  [],
+  []
+]
+
+// function to execute to make the board visible
+function displayBoard() {
+  console.clear()
+  board.map(function(pegs) {
+    console.log(`--- ${pegs}`)
+    return
+  })
+}
+
+
+
+
+// -- main game-play function -- // 
+
+
+function moveDiscs(start, dest) {
+  start--
+  dest--
```

Comment:
its a very bad practice to modify arguments without storing them into a new const.
```suggestion
  const adjustedStart = start--
  const adjustedDest = dest--
```
Its prone to bugs.